wallet/src/wallet.rs

```diff
@@ -1307,6 +1307,38 @@ mod tests {
         assert!(process_command(&config).is_err());
     }
 
+    #[test]
+    fn test_wallet_deploy() {
+        let mut pathbuf = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+        pathbuf.push("tests/fixtures/noop");
```

Comment:
nit: use `join()` to remove those `/` that won't work on some platforms (err, that we currently don't support)

Reply:
Ah, yes. I'll use PathBuf::push() to stay consistent with Jack's code.